velox/exec/Operator.h

```diff
@@ -129,6 +129,7 @@ struct OperatorStats {
   MemoryStats memoryStats;
 
   std::unordered_map<std::string, RuntimeMetric> runtimeStats;
+  std::set<int> driverIds;
```

Comment:
- std::unordered_set
- maintaining a set of driver IDs is a lot more expensive than maintaining a simple counter; any particular reason not to use a simpler counter?

Reply:
Fixed to use a simpler counter. Each driver updates the stats on close, so a counter should work.